FantasyUniverseTests/QuestionDataConverterTests.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  QuestionDataConverterTests.swift
+//  FantasyUniverseTests
+//
+//  Created by TarasPeregrinus on 27.03.2022.
+//
+
+import XCTest
+@testable import FantasyUniverse
+
+class QuestionDataConverterTests: XCTestCase {
```

Comment:
You can also test the mapping part, for example whether the answers or the `correct` answer are mapped correctly